CMakeLists.txt

```diff
@@ -20,30 +13,27 @@ add_library(gfx STATIC gfx.cpp)
 set(GFX_DXC_PATH ${CMAKE_CURRENT_SOURCE_DIR}/third_party/dxc)
 set(GFX_PIX_PATH ${CMAKE_CURRENT_SOURCE_DIR}/third_party/WinPixEventRuntime)
 
-set(GFX_DXC_PATH ${GFX_DXC_PATH} PARENT_SCOPE)
-set(GFX_PIX_PATH ${GFX_PIX_PATH} PARENT_SCOPE)
+if(NOT PROJECT_IS_TOP_LEVEL)
+	set(GFX_DXC_PATH ${GFX_DXC_PATH} PARENT_SCOPE)
+	set(GFX_PIX_PATH ${GFX_PIX_PATH} PARENT_SCOPE)
+endif()
```

Comment:
Avoid cmake warnings by only setting in parent scope when there actually is a parent scope